### ISSUE TITLE
Localisation

### ISSUE THREAD

**Issue by ronanh16** [odehir/ONT-Kit14_HIV1#1]
Dear Mr Hirotaka,
We read your paper on this protocol and I was wondering if you had a de-localised form of this repository that could be transferred and used on a different in-house system? We are interested in running this pipeline here in Sheffield and thought we'd check if you had a general use script before doing our own localisation. 
Many thanks,
Ronan, Sheffield University Department of Clinical Medicine

**Comment by odehir** [comment]
Dear Ronan

Thank you for reading our paper and having interest in our protcol. We are afraid that we don't  have the delocalized form of this repository. If you need our help, please feel free to contact us.
Best, Regards

Hirotaka

**Comment by odehir** [comment]
> Dear Mr Hirotaka, We read your paper on this protocol and I was wondering if you had a de-localised form of this repository that could be transferred and used on a different in-house system? We are interested in running this pipeline here in Sheffield and thought we'd check if you had a general use script before doing our own localisation. Many thanks, Ronan, Sheffield University Department of Clinical Medicine

Dear Ronan

Thank you for reading our paper and having interest in our protcol. We are afraid that we don't have the delocalized form of this repository. If you need our help, please feel free to contact us.
Best, Regards

Hirotaka

**Comment by ronanh16** [comment]
Thank you for your response, I will definitely get in touch if we need further support! 
Regards, Ronan

**Comment by ronanh16** [comment]
Hello again! I have successfully run the protocol in our system, but we were wondering you process the data to extract quasispecies and DRM abundance. Your article states that  "intrasample occupancies of the nucleotide and amino acid mutations at each position were analyzed by in-house programs". Are these publicly available?
Many thanks again,
Ronan

**Comment by odehir** [comment]
> Hello again! I have successfully run the protocol in our system, but we were wondering you process the data to extract quasispecies and DRM abundance. Your article states that "intrasample occupancies of the nucleotide and amino acid mutations at each position were analyzed by in-house programs". Are these publicly available? Many thanks again, Ronan

I'm so grad to hear from you that you have successfully run the protocol in your system. I would like you to confirm that CSV files with the suffix of "summary.csv" are outputted. The files may contain information on nucleotide bases or amino acids per position accoding to HXB2 position numbering for a  genome regeon, and is generated by "summary.pl" and the others found in this github site.
If you cannot find them,  please ask me again.

**Comment by ronanh16** [comment]
Yes we have these. If my interpretation is correct it shows every variation on each residue position and their frequency, but is their any way to deduce which variants are present in each quasispecies within the sample, and which are DRMs? The DRM question we can fix by extracting relevant mutations using HIVdb if it's not built into the protocol, but the quasispecies question would remain unanswered. 
Many thanks!

**Comment by odehir** [comment]
> Yes we have these. If my interpretation is correct it shows every variation on each residue position and their frequency, but is their any way to deduce which variants are present in each quasispecies within the sample, and which are DRMs? The DRM question we can fix by extracting relevant mutations using HIVdb if it's not built into the protocol, but the quasispecies question would remain unanswered. Many thanks!

Dear Ronan

As you may know, there are several interpretation algorithms for drug-resistance-assoiated mutations, like HIVdb, ANRS, and IAS-USA. Moreover, the algorithms have been updated continuously. Hence, we did not develop a script to automatically pick up drug-resistance-assoiated mutations.

> is their any way to deduce which variants are present in each quasispecies within the sample, 

As we have shown in our paper,  we can pick up some co-existent mutations within a given sample. Please find the files with the suffix of ".aa.csv", which contain amino acid profiles at each position for a sample. From information in them,  I think that you can extract linked mutations.

In contrast, it is still difficult for us to estimate sequences for each quasispieces, although we are trying. It is partially  because of still high error rate.

Best,

Hirotaka

**Comment by ronanh16** [comment]
Thank you for your insight! I'll take your advice on extracting linked mutations; I'll keep in touch if I develop anything promising. 
Best,
Ronan
